src/features/HomePage/components/Welcome.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+import { palette } from '@/components/variables';
+import Text from '@/components/Text';
+import { TextButton } from '@/components/Buttons';
+
+import type { UserRole } from '@/features/Authentication/userSlice';
+
+type Props = {
+  role: UserRole;
+};
+
+const Welcome = ({ role }: Props) => {
+  const { t } = useTranslation('home');
+
+  // REFACTOR
+  const navigation = {
+    admin: '/admin',
+    mentor: '/chat',
+    mentee: '/mentors',
+  };
```

Comment:
Refactor-comment? Idea is to get a navigation-object that can be shared between the nav-bar and this component?

Reply:
Oh I just forgot to remove that comment!